open-company/open-company-web
# Last section standing

Card: https://trello.com/c/QNkwf9i1

Abstract: https://projects.invisionapp.com/share/WNNVHKIT7XH

To test:
- start with at least 2 sections
- delete a section
- [x] do you see the section disappear from left nav and you are redirected to AP? Good
- click New
- [x] do you NOT see the deleted section in the sections list picker? Good
- now try to delete the last section
- [x] could you NOT delete it? Good
- [x] did you see the tooltip saying that you can't delete the last remaining section? Good
